README.md

```diff
@@ -155,10 +149,6 @@ This runs [Prettier][] to automatically format the entire codebase.
 
 This runs [Prettier][] on only your changed files. This is intended for development.
 
-### `yarn snyk`
```

Comment:
Heh, I guess that was a lie

Reply:
It may have done something since I think that would have run `node_modules/.bin/snyk`